edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/EdgeHubScopeDevice.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Http.Controllers
+{
+    using System.Collections.Generic;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Shared;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+
+    public class EdgeHubScopeDevice
```

Comment:
What is an EdgeHubScopeDevice? Can you pls add a comment?

Reply:
Filed bug to track this

[https://msazure.visualstudio.com/One/_workitems/edit/7649919](https://msazure.visualstudio.com/One/_workitems/edit/7649919)

---
In reply to: [451638847](https://github.com/Azure/iotedge/pull/3149#discussion_r451638847) [](ancestors = 451638847)